osu.Server.Spectator/Database/DatabaseAccess.cs

```diff
@@ -50,7 +50,7 @@ public async Task<bool> IsUserRestrictedAsync(int userId)
 
         public Task<multiplayer_room> GetRoomAsync(long roomId)
         {
-            return connection.QueryFirstOrDefaultAsync<multiplayer_room>("SELECT * FROM multiplayer_rooms WHERE category = 'realtime' AND id = @RoomID", new
+            return connection.QueryFirstOrDefaultAsync<multiplayer_room>("SELECT * FROM multiplayer_rooms WHERE type != 'playlists' AND id = @RoomID", new
```

Comment:
This was intended to be in another PR and was actually required for https://github.com/ppy/osu-web/pull/7965#discussion_r682385215. I've deployed to production already, but will just leave it to be merged as part of this PR since it seems like it's close to ready.